### ISSUE TITLE
Flux backbone tends to generate different results even with fixed input

### ISSUE THREAD

**Issue by Bili-Sakura** [River-Zhang/ICEdit#54]
### How to fix the output with same input

It seems `Flux.1 [Dev]` would yield greatly different results with the same input even **fixing the seed of generator**.  (I test without loading LoRA where it stills yield various result images.)
I wonder whether it is due to the undeterimined sampling strategy or the randomness of DiT itself?

#### Code Snipts

```python
from diffusers.utils import load_image, make_image_grid
import torch
import numpy as np
from PIL import Image

pre_event_img= <PIL.Image> ### 
PROMPT = <str> ### 
NUM_INFERENCE_STEPS=50
GUIDANCE_SCALE=7
SEED = 42
GENERATOR = torch.manual_seed(SEED)
```

```python
from diffusers import FluxFillPipeline
pipe = FluxFillPipeline.from_pretrained(
    "/home/models/black-forest-labs/FLUX.1-Fill-dev",
    torch_dtype=torch.float16,
)
pipe = pipe.to("cuda") 
# pipe.load_lora_weights("/home/models/sanaka87/ICEdit-MoE-LoRA/ICEdit-MoE-LoRA.safetensors")  
# pipe.load_lora_weights("/home/models/RiverZ/normal-lora/pytorch_lora_weights.safetensors")  
```

```python

width, height = pre_event_img.size

# Create diptych: left=original, right=original (to be edited)
combined_image = Image.new("RGB", (width * 2, height))
combined_image.paste(pre_event_img, (0, 0))
combined_image.paste(pre_event_img, (width, 0))

# Create mask: right half is 255, left half is 0
mask_array = np.zeros((height, width * 2), dtype=np.uint8)
mask_array[:, width:] = 255
mask = Image.fromarray(mask_array)

# Compose the instruction as in inference.py
instruction = f'A diptych with two side-by-side images of the same scene. On the right, the scene is exactly the same as on the left but {PROMPT}'

# Run the pipeline
result_image = pipe(
    prompt=instruction,
    image=combined_image,
    mask_image=mask,
    height=height,
    width=width * 2,
    guidance_scale=GUIDANCE_SCALE,  
    num_inference_steps=NUM_INFERENCE_STEPS,  #
    generator=GENERATOR,
).images[0]

# Crop to get only the edited (right) side
result_image = result_image.crop((width, 0, width * 2, height))
result_image
```
